src/main/resources/application.yml

```diff
@@ -0,0 +1,53 @@
+server:
+  servlet:
+    context-path: /api
+
+# Actuator configurations
+management:
+  endpoints:
+    web:
+      exposure:
+        include: all
+
+# Spring configurations
+spring:
+  datasource:
+    url: jdbc:mysql://localhost:3306/order_mgmt?useSSL=false&useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC
+    username: root
+    password: password
+  jpa:
+    hibernate:
+      ddl-auto: validate
+    show-sql: true
+  profiles:
+    active: local
+  application:
+    name: order-management
+  kafka:
+    producer:
+      bootstrap-servers: localhost:9092
+      client-id: orderManagementProducer
+      key-serializer: org.apache.kafka.common.serialization.StringSerializer
+      value-serializer: org.springframework.kafka.support.serializer.JsonSerializer
+      acks: all
+      retries: 3
+      batch-size: 16384
+      buffer-memory: 33554432
+      linger-ms: 1
+      enable-Idempotence: true
+      transactionalId: orderTransaction
+    consumer:
+      bootstrap-servers: localhost:9092
+      client-id: orderManagementConsumer
+      key-deserializer: org.springframework.kafka.support.serializer.ErrorHandlingDeserializer
+      value-deserializer: org.springframework.kafka.support.serializer.ErrorHandlingDeserializer
```

Comment:
If you are using `ErrorHandlingDeserializer` then you have handle few more stuffs on your `KafkaConsumerConfiguration` file. 
Read this article for how to do it https://www.confluent.io/blog/spring-kafka-can-your-kafka-consumers-handle-a-poison-pill/

Reply:
Hi Gagan, Thank you for your comment. I was trying few things with ErrorHandlingDeserilizer and later forgot to change it to StringSerializer. I have made changes to the configuration file. Please check. I am not able to send PR with changes.